web/test/pages/[urlSlug].test.tsx

```diff
@@ -220,4 +224,50 @@ describe("task page", () => {
     expect(subject.queryByTestId("post-onboarding-true-content")).not.toBeInTheDocument();
     expect(currentUserData().onboardingData.constructionRenovationPlan).toBe(false);
   });
+
+  describe("next and previous task buttons", () => {
+    const taskOne = generateTask({ urlSlug: "task-1" });
+    const taskTwo = generateTask({ urlSlug: "task-2" });
+    const taskThree = generateTask({ urlSlug: "task-3" });
+    beforeEach(() => {
+      useMockRoadmap({
+        steps: [
+          generateStep({
+            tasks: [taskOne],
+          }),
+          generateStep({
+            tasks: [taskTwo],
+          }),
+          generateStep({
+            tasks: [taskThree],
+          }),
+        ],
+      });
+    });
+
+    it("renders only Next Task button for first task", async () => {
+      subject = renderPage(taskOne);
+      expect(subject.getByText(TaskDefaults.nextTaskButtonText)).toBeInTheDocument();
+      expect(subject.queryByText(TaskDefaults.previousTaskButtonText)).not.toBeInTheDocument();
+    });
+    it("renders only Previous Task button at last task", async () => {
+      subject = renderPage(taskThree);
+      expect(subject.queryByText(TaskDefaults.nextTaskButtonText)).not.toBeInTheDocument();
+      expect(subject.getByText(TaskDefaults.previousTaskButtonText)).toBeInTheDocument();
+    });
```

Comment:
was this run through prettier? I'm surprised it passed through without adding newlines between tests. can we add that?

Reply:
also a newline directly above the beforeeach